### PR TITLE
add Kappa Ray 0.3.3 to CKAN

### PR DESCRIPTION
**Pull request by ec429**
The docs say I can't have a hyphen in <identifier>, so I've gone for 'kapparay' rather than 'kappa-ray' (the latter being the name of the folder under GameData).

### CODE REVIEW

File: kapparay/kapparay-0.3.3.ckan [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    "spec_version": 1,
+    "identifier"  : "kapparay",
+    "name"        : "Kappa Ray",
+    "author"      : [ "soundnfury" ],
+    "abstract"    : "Subject Kerbals and equipment to radiation",
+    "version"     : "0.3.3",
+    "ksp_version" : "1.0.5",
+    "license"     : "GPL-2.0",
+    "resources": {
+        "homepage": "http://forum.kerbalspaceprogram.com/threads/131575",
```
Comment (Olympic1) [inline kapparay/kapparay-0.3.3.ckan]: remove comma

File: kapparay/kapparay-0.3.3.ckan [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+    },
+    "download": "http://jttlov.no-ip.org/tar/kapparay-0.3.3.zip",
+    "depends": [
+        { "name": "ModuleManager" },
```
Comment (Olympic1) [inline kapparay/kapparay-0.3.3.ckan]: remove comma

File: kapparay/kapparay-0.3.3.ckan [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    ],
+
+    "install": [
+        {
+            "file"       : "kapparay-0.3.3/GameData/kappa-ray",
+            "install_to" : "GameData",
```
Comment (Olympic1) [inline kapparay/kapparay-0.3.3.ckan]: remove all the comma's from this point

**Comment by plague006** [comment]
> The docs say I can't have a hyphen in

The document you read is in error, would you mind posting a link so it can be corrected? You can indeed have hyphens in identifiers see: all ContractConfigurator contract packs.

**Comment by ec429** [comment]
plague06: https://github.com/KSP-CKAN/CKAN/wiki/Adding-a-mod-to-the-CKAN#other-host-guide says "It must contain only alphanumeric characters (no spaces, no punctuation, no special characters)".  Admittedly, some definitions of alphanumeric characters include _, but very few include -.  (Also, in the C-like family of languages, things called <identifier> are almost always defined as `[_a-zA-Z][_0-9a-zA-Z]*`)

But if hyphen is OK, I'll use it in next version of pull req.

Olympic1: will do.

**Comment by plague006** [comment]
@ec429 thanks, changes have been made to be more precise about allowed identifier names.

**Pull request closed without merging**